docs/guides/09-add-interactivity-and-events.md

```diff
@@ -1,5 +1,5 @@
 ## Add interactivity and events
-In this guide you will learn how to add user interactions to your CARTO VL visualization. By the end of this guide, you will have a better understanding of interactivity events (e.g. waiting for a layer to load or clicking on a feature) and how they can be used to make your visualization more dynamic and provide a richer experience for the end-user (e.g. changing the color of features on hover or click). You will also learn how to add common add-ons like *pop-ups* with additional information about the data being clicked on or hovered.
+In this guide you will learn how to add user interactions to your CARTO VL visualization. By the end of this guide, you will have a better understanding of [interactivity](https://carto.com/help/glossary/#interactivity) events (e.g. waiting for a layer to load or clicking on a feature) and how they can be used to make your visualization more dynamic and provide a richer experience for the end-user (e.g. changing the color of features on hover or click). You will also learn how to add common add-ons like *pop-ups* with additional information about the data being clicked on or hovered.
```

Comment:
I think that in this context it is better to rephrase it a bit (the old one was a little confusing too), as "waiting for a layer" is an event, but not an interactivity (as per the glossary definition).


```suggestion
In this guide you will learn how to add user interactions to your CARTO VL visualization. By the end of this guide, you will have a better understanding of events (e.g. waiting for a layer to load or clicking on a feature) and how they can be used to make your visualization more dynamic and also provide a richer experience for the end-user (e.g. changing the color of features on hover or click, thanks to our [interactivity](https://carto.com/help/glossary/#interactivity) features). You will also learn how to add common add-ons like *pop-ups* with additional information about the data being clicked on or hovered.
```

Reply:
👍